src/QSymbolicsBase/predefined.jl

```diff
@@ -106,6 +106,37 @@ symbollabel(::HGate) = "H"
 ishermitian(::HGate) = true
 isunitary(::HGate) = true
 
+abstract type AbstractRotGate <: AbstractSingleQubitGate end
+ishermitian(::AbstractRotGate) = false
+isunitary(::AbstractRotGate) = true
+isexpr(::AbstractRotGate) = true
+iscall(::AbstractRotGate) = true
+arguments(x::AbstractRotGate) = [x.θ]
+
+@withmetadata struct RotXGate <: AbstractRotGate
+    θ
+end
+operation(::RotXGate) = Rx
+head(::RotXGate) = :RotX
+children(x::RotXGate) = [:RotX, x.θ]
```

Comment:
This `head` (`:RotX`) does not seem to be something that exists. This (and for Y and for Z) should probably be `:RotXGate`